meshtastic/tests/test_mesh_interface.py

```
@@ -3658,3 +3728,227 @@ def _ack_permitted_callback(_packet: dict[str, Any]) -> None:
+
+        def _response_callback(packet: dict[str, Any]) -> None:
+            callback_calls.append(packet)
+
+        iface.responseHandlers[88] = ResponseHandler(
+            callback=_response_callback, ackPermitted=True
+        )
+
+        original_message_to_dict = google.protobuf.json_format.MessageToDict
+
+        def _message_to_dict_with_position_failure(
+            message: Any,
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `message`

(ANN401)